main.go

```diff
@@ -270,7 +276,39 @@ func (bridge *Bridge) Init() {
 	bridge.MatrixHandler = NewMatrixHandler(bridge)
 	bridge.Formatter = NewFormatter(bridge)
 	bridge.Crypto = NewCryptoHelper(bridge)
-	bridge.Metrics = NewMetricsHandler(bridge.Config.Metrics.Listen, bridge.Log.Sub("Metrics"), bridge.DB)
+	bridge.Metrics = NewMetricsHandler(bridge.Config.Metrics.Listen, bridge.Log.Sub("Metrics"), bridge.DB, bridge.PuppetActivity)
+}
+
+func (mh *Bridge) UpdateActivePuppetCount() {
+	if mh.Config.AppService.Limits.MinPuppetActiveDays == 0 {
```

Comment:
I'm a bit confused as to why 0 is not a valid value?
`1` day seems to only count them after 24 hours, but what if I want users to be counted immediately?
Currently, it shuts tracking off.

Reply:
Yeah, I was wondering about this. I think zero is likely a valid value, perhaps we should just always calculate this figure.

Reply:
Done